docs(windows): Initial commit - README.md (#5119)
keymanapp/keyman: windows/src/developer/README.md

# Keyman Developer for Windows

This is the current home for Keyman Developer except for the Lexical Model Compiler.

The Lexical Model Compiler is found at the root level /developer folder.

## inst - keymandeveloper.msi

Building and signing of the installation archive keymandeveloper.msi

## kmanalyze - kmanalyze.exe

A keyboard source analysis and automated regression test tool. Testing for logical errors of a keyboard.

## kcmpdll - kmcmpdll.dll & kmcmpdll.x64.dll

> This is the main source code for the keyboard complier it is packaged as dynamic linked library. The consumers will call 'CompileKeyboardFile'

Reviewer: ```suggestion
This is the main source code for the keyboard compiler, packaged as dynamic linked library. The consumers will call 'CompileKeyboardFile'
```

## kcframe.exe & kcframe.x64.exe

Located under kmcomp folder this is used for debugging in Visual Studio.  A command line program compiler for a keyboard file 'kmn' into a binary format 'kmx'. Has hardcoded default arguments.

## kmcomp - kmcomp.exe & kmcomp.x64.exe

> The main command line compiler for a keyboard. It compiles keyboards in either kmn or .kps format.

Reviewer: ```suggestion
The main command line compiler for a keyboard. It compiles keyboards (.kmn), packages (.kps), and projects (.kpj).
```
